docs/scarpet/api/Auxiliary.md

```diff
@@ -268,6 +268,14 @@ Example usages:
   // not a problem in apps
 </pre>
 
+### `item_display_name(item)`
+ Returns the name of the item as a Text Value. `item` should be a list of `[item_name, count, nbt]`, or just an item name.
```

Comment:
`/script run item_display_name(['iron_nugget', 0, null])` returns 'Air' instead of null or raising UNKNOWN_ITEM, and this piece of documentation says it would handle that. 

I assume the only reason to support item triples is to handle cases where items are renamed in nbt, correct?